README.md

```diff
@@ -45,7 +45,20 @@ The `ntripping` utility has the following usage:
             --url <url>             [default: na.skylark.swiftnav.com:2101/CRS]
 
 Different resources can be requested from different locations. By default, a San
-Francisco latitude, longitude, and height will be used.
+Francisco latitude, longitude, and height will be used. The data returned is in RTCM format.
+
+The `--url` must be formatted as such: `{username}:{passsword}@{area}.skylark.swiftnav.com:{port}/{mountpoint}`, 
+where `username` and `password` refer to the user's NTRIP username and password -- for Swift Skylark users
+this can be looked up from the Device Registry (internal to Swift). Internal Swift Skylark users
+can also lookup the various Skylark endpoints and mountpoints in our documentation.
+
+For example, if a user named John Doe has registered for a username of `john.doe` and password of `pa$$w0rd`
+and wants to query for OSR data at
```

Comment:
NTRIP returns sources of information from mountpoints - OSR data here is not an applicable term.